### PR TITLE
Takeover animation for using with react component

### PR DESCRIPTION
**Pull request by AndreasArledal**
- Added styling for animating the takeover
- Changed takeover title to more semantic h1
- Removed unnecessary .takeover-title class
screencast 2015-11-26 21-09-12

### CODE REVIEW

File: src/mnd-bootstrap/javascript/_takeover.scss [hunk 1]
```diff
@@ -142,3 +138,27 @@ Based on bootstrap modal.
 
 }
 
+// Animations added for using as react component
+.takeover-animation {
+  @include transition(opacity .15s ease-in-out, visibility 0s linear .15s);
```
Comment (mnd-hound) [inline src/mnd-bootstrap/javascript/_takeover.scss]: `.15` should be written with a leading zero as `0.15`

File: src/mnd-bootstrap/javascript/_takeover.scss [hunk 2]
```diff
@@ -142,3 +138,27 @@ Based on bootstrap modal.
...
+  display: block;
+  opacity: 0;
+  visibility: hidden;
+  z-index: $zindex-modal;
+
+  .takeover-content {
+    @include translate(0, -25%);
+    @include transition-transform(.3s ease-out);
```
Comment (mnd-hound) [inline src/mnd-bootstrap/javascript/_takeover.scss]: `.3` should be written with a leading zero as `0.3`

**Comment by AndreasArledal** [comment]
I'm not sure if the react example should be included. It feels good to have example of usages of components. But when we have components in another repo it doesn't feel quite right. What's your opinions?

**Comment by b2l** [comment]
I would say keep the React example.
I agree that it's a bit strange when the component is somewhere else.
There have been some prototype to have the components in the styleguide, so that we can have the documentation close by, and then import them in the different project.
I don't remember the status of this, @zoeesilcock may have some input.

**Comment by b2l** [comment]
Otherwise, looks good to me!

**Comment by AndreasArledal** [comment]
@b2l Yes, we're likely gonna work on it during lab days. Maybe we could do some syncing before regarding #26?

**Comment by jorkas** [comment]
I agree @b2l! We keep it and try to get the components into this repo! :shipit:

**Comment by b2l** [comment]
Sure @AndreasArledal There are still some work to do on this, but it was a working prototype.

**Pull request merged**